src/lua/discombobulate.lua

```diff
@@ -66,16 +73,9 @@ redis.call("set", exp_key, 1, "ex", ttl)
 redis.call(
     "xadd", strm_key, "*",
     "version", event_version,
-    "type", "discombobulator",
-    "user_id", source_id,
-    "guild_id", guild_id,
-    "amount", -offer
-)
-redis.call(
-    "xadd", strm_key, "*",
-    "version", event_version,
-    "type", "discombobulatee",
-    "user_id", target_id,
+    "type", "discombobulate",
+    "source_id", source_id,
+    "target_id", target_id,
```

Comment:
@Zymna what do you think about this change?

Reply:
Rationale behind this change is: emit only one event with all the info instead of two, so we can do [this](https://github.com/ass-a-service/ajobot/pull/85/files#diff-20e2eac0fc0e8221d9d8f32eafc80f220766a021ac6beb1f6138735404fee321R77)

Reply:
By doing this we're breaking [double-entry bookkeeping](https://en.wikipedia.org/wiki/Double-entry_bookkeeping).
I suggest adding additional parameters into the two events instead.